src/coreclr/jit/CMakeLists.txt

```diff
@@ -75,12 +75,16 @@ function(create_standalone_jit)
   if ((TARGETDETAILS_ARCH STREQUAL "x64") OR (TARGETDETAILS_ARCH STREQUAL "arm64") OR ((TARGETDETAILS_ARCH STREQUAL "x86") AND NOT (TARGETDETAILS_OS STREQUAL "unix")))
     target_compile_definitions(${TARGETDETAILS_TARGET} PRIVATE FEATURE_SIMD)
     target_compile_definitions(${TARGETDETAILS_TARGET} PRIVATE FEATURE_HW_INTRINSICS)
+    target_compile_definitions(${TARGETDETAILS_TARGET} PRIVATE FEATURE_MASKED_SIMD)
+    target_compile_definitions(${TARGETDETAILS_TARGET} PRIVATE FEATURE_MASKED_HW_INTRINSICS)
```

Comment:
I don't think we really need both of these.

`FEATURE_SIMD` itself is somewhat of a legacy concept and `TYP_MASK` will only ever be produced in relation to `FEATURE_HW_INTRINSIC` code.

I think we can just have one define here which XARCH and ARM64 both define.

Reply:
> I think we can just have one define here

Done.